requirements.txt

```diff
@@ -4,3 +4,4 @@ pydantic
 typing-extensions
 appdirs
 requests
+importlib-resources
```

Comment:
It looks like there's a [`importlib.resources` module](https://docs.python.org/3/library/importlib.resources.html) that has been available as part of the std libs since python v3.7. Since python 3.6 has reached end-of-life and this theme requires v3.8+, does this mean we don't need a new external dependency? Is there some advantage to using the thrid-party [`importlib_resources` lib](https://importlib-resources.readthedocs.io/en/latest/using.html)?

Reply:
After some more thought, I don't think the cached default fonts needs to be an importable resource. We could just use 

```py
# from google_fonts.py or external_resource_cache.py
pathlib.PurePath(__file__).parent / f"{default_font_location}" / f"{default_font_name}"
```

The `importlib.resources` module seems designed to let external API easily find a non-python asset from our pkg.

Reply:
`importlib-resources` is added due to the fact that Python versions below 3.9 do not have `files` in `importlib.resources`. But sure, I can do a workaround that will not use `importlib-resources` dependency